huggingface-gpt2/ort_addon/ort_supplement/src/transformers/trainer_ort.py

```diff
@@ -0,0 +1,578 @@
+import json
+import time
+import logging
+import os
+import random
+import re
+import shutil
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Callable, Dict, List, NamedTuple, Optional, Tuple
+
+import numpy as np
+import torch
+from torch import nn
+from torch.utils.data.dataloader import DataLoader
+from torch.utils.data.dataset import Dataset
+from torch.utils.data.distributed import DistributedSampler
+from torch.utils.data.sampler import RandomSampler
+from tqdm import tqdm, trange
+
+import onnxruntime
+from onnxruntime.capi.ort_trainer import ORTTrainer, IODescription, ModelDescription
+from onnxruntime.capi.ort_trainer import LossScaler
+
+from .data.data_collator import DataCollator, DefaultDataCollator
+from .modeling_utils import PreTrainedModel
+from .training_args import TrainingArguments
+from .trainer import PredictionOutput, TrainOutput, EvalPrediction, set_seed
+
+from azureml.core.run import Run
+# get the Azure ML run object
+run = Run.get_context()
+
+try:
+    from apex import amp
+
+    _has_apex = True
```

Comment:
nit: sorry didnt catch this earlier. we can remove all apex related stuff in here.